R/class-plotdata.R

```diff
@@ -40,6 +41,16 @@ newPlotdata <- function(.dt = data.table(),
   lon <- veupathUtils::findColNamesFromPlotRef(variables, 'longitude')
 
   isEvil <- ifelse(evilMode %in% c('allVariables', 'strataVariables'), TRUE, FALSE)
+  collectionVarMetadata <- veupathUtils::findCollectionVariableMetadata(variables)
+  isOverlayCollection <- ifelse(is.null(collectionVarMetadata), FALSE, ifelse(collectionVarMetadata@plotReference@value %in% c('overlay'), TRUE, FALSE))
```

Comment:
is there an R power user reason for checking `value %in% c('overlay')` as opposed to `identical(value, 'overlay')`?